docker/alpine-non-root.dockerfile

```
@@ -0,0 +1,27 @@
+#example build command:
+#start in the root dir
+#podman build -t non-root -f docker/alpine-non-root.dockerfile .
+#create build image
+FROM node:16-alpine AS build
+#standardverzeichnis
+WORKDIR /app
+
+#or download
+COPY . .
+#--production do not work bc : vite not found
+#RUN npm install --production
+RUN npm install
+RUN npm run build
+
+#create release image
+FROM node:16-alpine AS release
+WORKDIR /app
+USER node
+#improve this
+COPY --from=build /app /app
+
+EXPOSE 3001
```

Move EXPOSE before copy to use little bit more cache when building locally (about 1s less)
```suggestion
EXPOSE 3001
#improve this
COPY --from=build /app /app

```